src/Components/BooksInputs.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/booksSlice';
+
+const BooksInputs = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <div>
+      <form onSubmit={(e) => {
+        e.preventDefault();
+        const title = e.target.elements.title.value;
+        const author = e.target.elements.author.value;
```

Comment:
- [ ] as the previous reviewer suggested you should use the local state `useState`  that will get the text `onChange` event 
using the dom element to get the value is not good practice and as you know one of the requirements is to keep you code safe and clean and not redundant so please make sure to do that

Reply:
@youmari,
thanks for your good and valuable feedback.
I will consider your comments and follow the instruction thanks again.